src/pages/index.astro

```diff
@@ -66,11 +75,13 @@ import { SITE_TITLE, SITE_DESCRIPTION } from "../consts";
   <body>
     <Header />
     <main>
-      <div class="hero">
-        <img src="burger.svg" alt="" class="burger" />
-        <h1 class="serving">Serving up</h1>
-        <p class="tasty">TASTY BYTES</p>
-        <h1 class="since">since 2014</h1>
+      <div id="content-start" class="hero">
+        <img src="burger.svg" alt="A cartoon cheeseburger with a flag sticking out" class="burger" />
+        <h1>
+          <span class="serving">Serving up</span>
+          <span class="tasty">TASTY BYTES</span>
+          <span class="since">since 2014</span>
```

Comment:
Can I get all of these to read in one sentence if they're lined up on the same line? Working on that right now.

If I can't find a way to do this, then this is fine.